hazelcast/src/main/java/com/hazelcast/jet/impl/submitjob/validator/SubmitJobParametersValidator.java

```diff
@@ -14,34 +14,47 @@
  * limitations under the License.
  */
 
-package com.hazelcast.jet.impl;
+package com.hazelcast.jet.impl.submitjob.validator;
 
 import com.hazelcast.jet.JetException;
-import com.hazelcast.jet.SubmitJobParameters;
+import com.hazelcast.jet.impl.SubmitJobParameters;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.List;
 import java.util.Objects;
 
 public class SubmitJobParametersValidator {
```

Comment:
Can we make all the methods of this class static so that we don't have to create an object for validation?

Reply:
OK. Revision : 0c495e5f595a44d0301bec31d9ea9935c8ff3b57